### PR TITLE
CGO#22 - Travis CI

### PR DESCRIPTION
**Pull request by Clearedkinkajou**
* Created a Travis CI config file that sets up the entire flutter environment

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (DBatistaFerreira) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: Hehe, thank you for pushing this to develop

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (DBatistaFerreira) [inline .github/ISSUE_TEMPLATE/user-story.md:1]: This too ^_^

File: codecov.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+codecov:
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```
Comment (DBatistaFerreira) [inline codecov.yml:7]: So we need 70% coverage?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+language: generic
```
Comment (DBatistaFerreira) [inline .travis.yml]: So what are the steps that travis does for our application from this file?

File: .gitignore [hunk 5]
```diff
@@ -27,6 +27,7 @@
 .pub-cache/
 .pub/
 build/
```
Comment (DBatistaFerreira) [inline .gitignore:29]: This will generate coverage reports or something else?

File: codecov.yml (hunk 3 above)
Comment (Clearedkinkajou) [reply codecov.yml:7]: no and yes, I have it set to not fail for now. Once we get tests going properly I will activate the code coverage rules

Comment (DBatistaFerreira) [reply codecov.yml:7]: Create a devops ticket in the backlog for activation of code coverage rules so we can still push this then :)

File: .travis.yml (hunk 4 above)
Comment (Clearedkinkajou) [reply .travis.yml]: Right now it install flutter and dependencies, then it runs all the tests and creates a coverage report, then it reads the coverage report and always passes. Finally it sends a notification to discord on the build status

Comment (DBatistaFerreira) [reply .travis.yml]: Awesome

File: .gitignore (hunk 5 above)
Comment (Clearedkinkajou) [reply .gitignore:29]: yeah, so we need to make sure we don't commit coverage reports

Comment (DBatistaFerreira) [reply .gitignore:29]: Sounds good. Note it to the team

**Review by DBatistaFerreira — APPROVED** [review]

File: codecov.yml (hunk 3 above)
Comment (Clearedkinkajou) [reply codecov.yml:7]: done #37

**Pull request merged**